.ci/eclipse-compiler-javac.sh

```diff
@@ -7,8 +7,8 @@ if [ -z "$1" ]; then
     exit 1
 fi
 
-ECJ_JAR="ecj-4.11.jar"
-ECJ_MAVEN_VERSION="R-4.11-201903070500"
+ECJ_JAR="ecj-4.17.jar"
+ECJ_MAVEN_VERSION="R-4.17-202009021800"
```

Comment:
please remove this update from this commit.

Reply:
The build will fail as `R-4.11-201903070500` was removed from the repo:
http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse/eclipse/downloads/drops4/

```
  | R-4.15-202003050155/ | 2020-03-18 14:29 | - |  
  | R-4.16-202006040540/ | 2020-08-06 09:42 | - |  
  | R-4.17-202009021800/ | 2020-09-16 16:00 | - |  
  | index.html | 2015-03-16 23:21 | 60 |  
```

Reply:
ok, lets keep it

Reply:
done

Reply:
Build log: https://checkstyle.semaphoreci.com/jobs/ee2aa9b3-6c6e-479a-a9a1-0bee0431808e#L458

Reply:
@pbludov , sorry for confusion, I meant, let keep version update in this commit.
or please keep eclipse for now in travis and we will move it separate PR.

Reply:
We can merge https://github.com/checkstyle/checkstyle/pull/8840 first and restart this build. These PR has no conflict.